samples/int/nc.bas

```diff
@@ -0,0 +1,121 @@
+' -- #FUJINET NETCAT Example --
+
+' Default unit # for connection
+CONN=2
+MODE=12
+TRANS=0
+
+' URL for connection
+URL$=""
+
+' RX Buffer
+DIM BUF(8192) BYTE
+
+' Procedures '''''''''''''''''''''''''
+
+PROC BANNER
+
+PRINT "** NETCAT IN FASTBASIC **"$9B
+ENDPROC
+
+PROC GETCONN
+  DO 
+
+    INPUT "URL: ",URL$
+
+    C$=URL$[1,2]
+
+    IF C$="N:"
+      EXIT
+    ENDIF 
+
+  LOOP
+
+  INPUT "TRANS (0=none, 1=CR, 2=LF, 3=CR/LF): ",TRANS
+  
+ENDPROC
+
+PROC INTCLR
+  POKE $D302, PEEK($D302) & 127
+ENDPROC
+
+PROC CONNECT
+
+  PRINT "Connecting to:"
+  PRINT URL$
+
+  NOPEN CONN,MODE,TRANS,URL$
+  @INTCLR
```

Comment:
I don't think you need the "INTCLR", as you are not using (nor reading) the interrupt status here.